woocommerce-services.php

```diff
@@ -1811,17 +1811,25 @@ public function add_plugin_action_links( $links ) {
 		 */
 		private static function _has_any_labels_db_check() {
 			global $wpdb;
+
 			// If $wpdb is not available yet, assume no labels (safer default).
-			if ( ! is_object( $wpdb ) || ! isset( $wpdb->postmeta ) ) {
+			if ( ! is_object( $wpdb ) ) {
 				return false;
 			}
+
+			$meta_table_to_check = $wpdb->prefix . 'wc_orders_meta';
```

Comment:
Why not use `OrderUtil::get_table_for_order_meta();` from `Automattic\WooCommerce\Utilities\OrderUtil` as in [here](https://github.com/woocommerce/woocommerce-shipping/blob/trunk/src/Migration/LegacyLabelMigrator.php#L113)?

Reply:
The meta table is either `$wpdb->prefix . 'wc_orders_meta'` or  `$wpdb->postmeta`

Reply:
oh, I wasn't aware of that function. Yeah, that works nicely, thanks for letting me know!